src/documents/serialisers.py

```diff
@@ -216,6 +220,18 @@ def get_archived_file_name(self, obj):
         else:
             return None
 
+    def update(self, instance, validated_data):
+        if "created_date" in validated_data and "created" not in validated_data:
+            new_datetime = datetime.datetime.combine(
+                validated_data.get("created_date"),
+                datetime.time(0, 0, 0, 0, tz.gettz(settings.TIME_ZONE)),
```

Comment:
I think zoneinfo is the preferred "modern" method for getting the tzinfo

Reply:
Totally defer on this, obviously. I got this from https://github.com/paperless-ngx/paperless-ngx/blob/9a1bd9637cbef0e7318a3c61cae14805c6510cea/src/documents/tests/test_date_parsing.py but happy to change if thats better

Reply:
Cool yea did some reading, thanks